jetty-core/jetty-fcgi/jetty-fcgi-server/src/main/java/org/eclipse/jetty/fcgi/server/internal/ServerFCGIConnection.java

```diff
@@ -206,7 +206,7 @@ else if (read == 0)
         catch (Exception x)
         {
             if (LOG.isDebugEnabled())
-                LOG.debug("Unable to fill endpoint", x);
+                LOG.atDebug().setCause(x).log("Unable to fill endpoint");
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.